client/app/bundles/course/assessment/pages/AssessmentIndex/translations.ts

```diff
@@ -0,0 +1,133 @@
+import { defineMessages } from 'react-intl';
+
+const translations = defineMessages({
+  newAssessment: {
+    id: 'course.assessment.newAssessment',
+    defaultMessage: 'New Assessment',
+  },
+  creationSuccess: {
+    id: 'course.assessment.create.success',
+    defaultMessage: 'Assessment was created.',
+  },
+  creationFailure: {
+    id: 'course.assessment.create.fail',
+    defaultMessage: 'Failed to create assessment.',
+  },
+  createAsDraft: {
+    id: 'course.assessment.create.createAsDraft',
+    defaultMessage: 'Create As Draft',
+  },
+  openingSoon: {
+    id: 'course.assessments.index.openingSoon',
+    defaultMessage: 'Opening soon',
+  },
+  unlockableHint: {
+    id: 'course.assessments.index.unlockableHint',
+    defaultMessage: 'Unlock this assessment by completing its requirements.',
+  },
+  unavailable: {
+    id: 'course.assessments.index.unavailable',
+    defaultMessage: 'Unavailable',
+  },
+  unavailableHint: {
+    id: 'course.assessments.index.unavailableHint',
+    defaultMessage:
+      'You cannot attempt this assessment because you are not a user in this course.',
+  },
+  title: {
+    id: 'course.assessments.index.title',
+    defaultMessage: 'Title',
+  },
+  autograded: {
+    id: 'course.assessments.index.autograded',
+    defaultMessage: 'Autograded',
+  },
+  passwordProtected: {
+    id: 'course.assessments.index.passwordProtected',
+    defaultMessage: 'Password-protected',
+  },
+  hasPersonalTimes: {
+    id: 'course.assessments.index.hasPersonalTimes',
+    defaultMessage: 'Has personal times',
+  },
+  hasPersonalTimesHint: {
+    id: 'course.assessments.index.hasPersonalTimesHint',
+    defaultMessage:
+      "Timings for this assessment will automatically be adjusted based on students' learning rate.",
+  },
+  affectsPersonalTimes: {
+    id: 'course.assessments.index.affectsPersonalTimes',
+    defaultMessage: 'Affects personal times',
+  },
+  affectsPersonalTimesHint: {
+    id: 'course.assessments.index.affectsPersonalTimesHint',
+    defaultMessage:
+      'Completion of this assessment may affect the timings for subsequent assessments.',
+  },
+  exp: {
+    id: 'course.assessments.index.exp',
+    defaultMessage: 'EXP',
+  },
+  bonusExp: {
+    id: 'course.assessments.index.bonusExp',
+    defaultMessage: 'Bonus',
+  },
+  neededFor: {
+    id: 'course.assessments.index.neededFor',
+    defaultMessage: 'Needed for',
```

Comment:
Requirement for?

Reply:
I changed it to `Needed for` because it's shorter, then it doesn't take up so much horizontal space in that column. Since I also limited the requirements to at most 4 circles (3 achievements + 1 `+N` remaining achievements), `Needed for`'s width fits perfectly with the 'maximum' width of the column.